types/telegram-web-app/index.d.ts

```diff
@@ -635,103 +635,145 @@ export type FileDownloadRequestedCallback = (eventData: {
 }) => void;
 
 /**
- * Web Apps can adjust the appearance of the interface to match the Telegram
- * user's app in real time. This object contains the user's current theme
- * settings:
+ * Mini Apps can adjust the appearance of the interface to match the Telegram user's app in real time.
+ * This object contains the user's current theme setting
  */
 export interface ThemeParams {
     /**
+     * *Optional.*
```

Comment:
I don't think it makes sense to mention this in the docs, given that the types already reflect whether it's optional or not